apps/explorer/src/components/orders/BridgeDetailsTable/styled.ts

```
@@ -0,0 +1,115 @@
+export const NetworkName = styled.span`
+  margin-left: 0.5rem;
+  color: ${Color.explorer_grey};
+`
+
+// RefundStatus styled components
+export const RefundStatusText = styled.span<{ status: RefundStatusType }>`
+  color: ${({ status }) => {
+    switch (status) {
+      case RefundStatusEnum.COMPLETED:
+        return Color.explorer_green1
+      case RefundStatusEnum.NOT_INITIATED:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)